sdks/python/apache_beam/coders/coders.py

```diff
@@ -1014,6 +1015,7 @@ def to_runner_api_parameter(self, context):
     else:
       return super(TupleCoder, self).to_runner_api_parameter(context)
 
+  @staticmethod
   @Coder.register_urn(common_urns.coders.KV.urn, None)
   def from_runner_api_parameter(unused_payload, components, unused_context):
     return TupleCoder(components)
```

Comment:
Should `ParamWindowedValueCoder.from_runner_api_parameter` also get the `@staticmethod` decorator?

Reply:
fixed